cmd/plugins/topology-aware/policy/pod-preferences.go

```diff
@@ -153,6 +157,36 @@ func sharedCPUsPreference(pod cache.Pod, container cache.Container) (bool, bool)
 	return preference, true
 }
 
+// cpuPrioPreference returns the CPU priority preference for the given container
+// and whether the container was explicitly annotated with this setting.
+func cpuPrioPreference(pod cache.Pod, container cache.Container, fallback cpuPrio) (cpuPrio, bool) {
```

Comment:
what's the use of the bool return value here? I see that you are ignoring the second return value when calling `cpuPrioPreference`, so I am wondering if we should rather drop it.

Reply:
Good point, and I will drop it if in the end of my still ongoing iteration it still not being used. 

But the idea with the second bool is that it tells whether the preference was explicit (indicated by the user) or implicit. I am considering a change so that if the preference is implicit, and there is no default priority set y the user (or it is set to none), then policy would ask for HighPrio CPUs, if an exclusive allocation request is for a single core or two cores on a hyperthreaded system.

Reply:
Updated.